user-service/controller/user-controller.js

```diff
@@ -9,8 +9,13 @@ export async function createUser(req, res) {
             if (resp.err) {
                 return res.status(400).json({message: 'Could not create a new user!'});
             } else {
-                console.log(`Created new user ${username} successfully!`)
-                return res.status(201).json({message: `Created new user ${username} successfully!`});
+                if (resp) {
+                    console.log(`Created new user ${username} successfully!`)
+                    return res.status(201).json({message: `Created new user ${username} successfully!`});
+                } else {
+                    console.log(`${username} already exists!`)
+                    return res.status(400).json({message: `${username} already exists!`});
+                }

```

Comment:
maybe could consider using else if here for better coding style

Reply:
Sure. Will make the change